HowToREST.md

```diff
@@ -0,0 +1,135 @@
+# How to Call Google APIs: REST Edition
+
+_For many developers, the easiest way to call a Google API is with one of our
+client libraries. But occasionally someone may prefer to make API calls
```

Comment:
Perhaps add a link to https://cloud.google.com/apis/docs/cloud-client-libraries?

Reply:
adding, thanks!